packages/rocketchat-ui-master/master/main.coffee

```diff
@@ -4,7 +4,7 @@ Template.body.onRendered ->
 	clipboard = new Clipboard('.clipboard')
 
 	$(document.body).on 'keydown', (e) ->
-		if e.keyCode is 80 and (e.ctrlKey is true or e.metaKey is true) and e.shiftKey is false
+		if e.keyCode is 80 or e.keyCode is 75 and (e.ctrlKey is true or e.metaKey is true) and e.shiftKey is false
```

Comment:
This `if` is not right 😄 
Did you try to hit only the key `p` to see what happens?

The correct code is:
```
if (e.keyCode is 80 or e.keyCode is 75) and (e.ctrlKey is true or e.metaKey is true) and e.shiftKey is false
```
or
```
if e.keyCode in [80, 75] and (e.ctrlKey is true or e.metaKey is true) and e.shiftKey is false
```